src/sync/pull-request.ts

```diff
@@ -57,18 +61,25 @@ export default async function(
 	const asyncTags:any[] = [];
 	// TODO: use graphql here instead of rest API
 	await statsd.asyncTimer(
+		// Retry up to 6 times pausing for 10s, for *very* large repos we need to wait a while for the result to succeed in dotcom
 		async () => {
-			({
-				data: edges,
-				status,
-				headers
-			} = await github.pulls.list({
-				...vars,
-				state: "all",
-				sort: "created",
-				direction: "desc"
-				// Retry up to 6 times pausing for 10s, for *very* large repos we need to wait a while for the result to succeed in dotcom
-			}));
+			(
+
+				{
+					data: edges,
+					status,
+					headers
+				} = useNewGHClient?
```

Comment:
If `useNewGHClient` we'll use an old one here

Reply:
I'm not a fan of this syntax.  I'd rather you separate the tertiary operator result into it's own const before doing an object deconstruction on it

Reply:
So most likely, this syntax will go away when the featureFlag is cleaned up